resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -79,12 +79,25 @@ private[spark] object KubernetesUtils extends Logging {
     opt2.foreach { _ => require(opt1.isEmpty, errMessage) }
   }
 
+  def getLocalTemplateFile(path: String, conf: SparkConf): File = {
```

Comment:
@dongjoon-hyun how about leveraging utils such as `DependencyUtils.downloadFile` that is used for Spark submit?

Reply:
Sure, I'll revise like that. In that case, we can remove the new test case for `getLocalTemplateFile`, too. Thanks.

Reply:
@attilapiros . The test case will be removed. So, I believe it resolved your concern, too. Is it okay for you?

Reply:
Sure, that is fine for me.